Shared/Samples/Find route in mobile map package/FindRouteInMobileMapPackageView.swift

```diff
@@ -0,0 +1,204 @@
+    struct MobileMapListView: View {
+        /// The mobile map package containing the maps.
+        let mapPackage: MobileMapPackage
+        
+        var body: some View {
+            // Create a list row for each map in the map package.
+            ForEach(mapPackage.maps.enumeratedArray(), id: \.offset) { (offset, map) in
+                let mapName = map.item?.name.titleCased ?? "Map \(offset + 1)"
+                
+                // The navigation link to the map.
+                NavigationLink {
+                    Group {
+                        if let locatorTask = mapPackage.locatorTask {
+                            MobileMapView(map: map, locatorTask: locatorTask)
+                        } else {
+                            MapView(map: map)
+                        }
+                    }
+                    .navigationTitle(mapName)
+                } label: {
+                    HStack {
+                        // The image of the map for the row.
+                        Image(uiImage: map.item?.thumbnail?.image ?? UIImage())
+                            .resizable()
+                            .scaledToFit()
+                            .frame(height: 50)
+                            .overlay {
+                                // The symbols indicating the map's functionality.
+                                VStack {
+                                    HStack {
+                                        if !map.transportationNetworks.isEmpty {
+                                            // The symbol indicating whether the map can route.
+                                            Image(systemName: "arrow.triangle.turn.up.right.circle")
+                                        }
+                                        Spacer()
+                                        if mapPackage.locatorTask != nil {
+                                            // The symbol indicating whether the map can geocode.
+                                            Image(systemName: "mappin.circle")
+                                        }
+                                    }
+                                    .padding(2)
+                                    Spacer()
+                                }
+                            }
+                        
+                        Text(mapName)
+                    }
+                }
+            }
+        }
+    }
+}
+
+private extension Collection {
+    /// Enumerates a collection as an array of (n, x) pairs, where n represents a consecutive integer
+    /// starting at zero and x represents an element of the collection..
+    /// - Returns: An array of pairs enumerating the collection.
+    func enumeratedArray() -> [(offset: Int, element: Self.Element)] {
+        return Array(self.enumerated())
+    }
+}
+
+private extension String {
+    /// A copy of a camel cased string broken into words with capital letters.
+    var titleCased: String {
```

Comment:
While this is a good attention to detail, I don't think it adds too much value as a mmpk can have title in languages other than en-US. Maybe just display its raw title.